src/main/java/com/gengo/client/GengoClient.java

```diff
@@ -352,9 +363,143 @@ public JSONObject getTranslationJobs() throws GengoException
      */
     public JSONObject getTranslationJobs(List<Integer> ids) throws GengoException
     {
-        String url = baseUrl + "translate/jobs/";
-        url += join(ids, ",");
-        return call(url, HttpMethod.GET);
+        return getTranslationJobs(ids, null, null, null);
+    }
+
+    /**
+     * Get specific translation jobs.
+     * @param ids a list of job ids to retrieve
+     * @param status one of the JobStatus
+     * @param timestampAfter Unix epoch seconds from which to filter submitted jobs
+     * @param count limit count of jobs(default 10, maximum 200)
+     * @throws GengoException
+     */
+    public JSONObject getTranslationJobs(List<Integer> ids, JobStatus status, Integer timestampAfter, Integer count) throws GengoException
+    {
+        String url = getBaseUrl() + "translate/jobs/";
+        JSONObject data = new JSONObject();
+        if (ids != null && ids.size() > 0) {
+            url += join(ids, ",");
+        }
+        if (status != null) {
+            try {
+                data.put("status", status.getStatusString());
+            } catch (JSONException e) {
+                throw new GengoException(e.getMessage(), e);
+            }
+        }
+        if (timestampAfter != null && timestampAfter > 0) {
+            try {
+                data.put("timestamp_after", timestampAfter);
+            } catch (JSONException e) {
+                throw new GengoException(e.getMessage(), e);
+            }
+        }
+        if (count != null && count > 0) {
+            try {
+                data.put("count", count);
+            } catch (JSONException e) {
+                throw new GengoException(e.getMessage(), e);
+            }
+        }
+        return call(url, HttpMethod.GET, data);
+    }
+
+    /**
+     * Set revision comment for each jobs.
+     * @param jobs each Entry consists of job ID and comment text
+     * @throws GengoException
+     */
+    public JSONObject reviseTranslationJobs(HashMap<Integer, String> jobs) throws GengoException
```

Comment:
@yugui Hmm, maybe I should update the Javadoc instead of the function name. Because the function is actually to revise jobs. However I realized that the Javadoc message is kinda different from responsibility. Does it make sense to you?